### ISSUE TITLE
Uninstall / remove MUPX instructions?

### ISSUE THREAD

**Issue by andrewash** [arunoda/meteor-up-legacy#880]
I'd like to uninstall MUPX from my server safely so I can use MUP instead.
I'm not sure how to undo what 'mupx setup' did.

Could you provide instructions?

FYI, the reason I'm reverting from MUPX to MUP is because my app reads and writes to the filesystem (see https://github.com/arunoda/meteor-up/issues/710)

**Comment by MasterJames** [comment]
There's a new version porting over at kadirahq maybe see what new solutions exist.
Basically the idea in the future is you spin up new instances always and often so Uninstaller is usually not considered.
What you want to observe is how the Mongo docker mounts a folder from the target and so you would like to add docker volume mount folders to the meteor docker container run script.

https://github.com/arunoda/meteor-up/blob/mupx/templates/linux/start.sh

In here you'd add a line to include you volume mount point where it does docker run .
So a custom PR or you can hack the npm install folder too.
Also new in the kadirahq version is the ability to make custom docker images so maybe there's something there about alternate volumes to mount or it would make a nice feature request.
I'm guessing there's a link on removal somewhere. Basically SSH into the target and follow some other remove docker instructions you'll note the Mongo mount point 'setup' creates in the docker run for Mongo if it's being installed by mupx in the volume mount point for it /var/lib/mongodb  or something like that i'll edit this if different.
Okay there's also /opt/$APPNAME

**Comment by andrewash** [comment]
Thank you for the FYI; I appreciate your taking the time to reply.
In the end I tracked down the running services and removed them via dpkg, along with their configuration files. 

The staging server I'm using for this project doesn't belong to me and isn't setup for this sort of rapid deployment (although that's what I tend to use for my own servers). 

From there I was able to re-deploy using mup (instead of mupx) and it's been working well.
